corehq/messaging/util.py

```diff
@@ -61,6 +65,17 @@ def increase_total_case_count(self, value):
         self.client.incr(self.total_key, delta=value)
         self.client.expire(self.total_key, self.key_expiry)
 
+    def cancel(self):
+        """Cancel the currently running task
+
+        This does nothing unless the task checks if it has been canceled.
+        """
```

Comment:
It took me a minute to digest this comment.  What do you think of something like:
```python
"""Mark the current task as canceled

The task is responsible for checking to see if it has been canceled and acting accordingly
"""
```